movement/watch_faces/complication/ratemeter_face.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022 Joey Castillo
```

Comment:
Could you change the name here to reflect your name? Just want to make sure that individuals' contributions are attributed properly (and that you're down with licensing yout watch face under MIT terms).

Reply:
updated - thanks! PTAL